dev-proxy/CommandHandlers/JwtCommandHandler.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.IdentityModel.Tokens;
+using System.Globalization;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using System.Security.Principal;
+
+namespace Microsoft.DevProxy.CommandHandlers;
+
+public static class JwtCommandHandler
+{
+    public static void CreateToken(string scheme, string name, IEnumerable<string> audience, string issuer, IEnumerable<string> roles, IEnumerable<string> scopes, IEnumerable<string> claims, double validFor)
+    {
+        var jwtIssuer = new JwtIssuer(
+            issuer,
+            RandomNumberGenerator.GetBytes(32)
+        );
+
+        var options = new JwtCreatorOptions(
+            scheme,
+            name,
+            audience,
+            issuer,
+            DateTime.UtcNow,
+            DateTime.UtcNow.AddHours(validFor),
+            roles,
+            scopes,
+            ParseClaims(claims)
+        );
+
+        var jwtToken = jwtIssuer.Create(options);
+
+        var jwt = Jwt.Create(
+            scheme,
+            jwtToken,
+            new JwtSecurityTokenHandler().WriteToken(jwtToken),
+            options.Scopes,
```

Comment:
Do we want to align with Entra ID which uses the `scp` claim for scopes? Right now, scopes are put in the `scope` claim in the generated token.